### ISSUE TITLE
[Feature Request] Handling options for "On clip creation"

### ISSUE THREAD

**Issue by marvnrawley** [crowbartools/Firebot#1582]
**Describe the solution you'd like**

Essential handling options for "On clip creation" are suggested to be
- Add event trigger for "When clip created"
- Parse the URL to the clip allowing an effect to play the newly created clip automatically on the overlay (possibly subject to the person who created the clip being in a given user group i.e. mods/VIPs/trusted editors etc are permitted to autoplay new clips)
- Ensure that integrations with discord for example allow the clip URL to be written in a channel with a custom message

Nice to have handling options 
- Event effect for firebot app to automatically download the clip to the local computer
- Event effect for automated submission to a YT channel

**Additional context**

There are bots which do ONLY the above, would be handy to integrate the features into firebot itself.

**Comment by iamdadmin** [comment]
This is probably blocked from upstream lack of PubSub in the API https://twitch.uservoice.com/forums/310213-developers/suggestions/40616533-clips-webhook - would have to poll instead. I may look at setting up a poll-based option, but not sure if Firebot as a direction wants to be doing that.

**Comment by zunderscore** [comment]
EventSub has no event that fires when clips are created. There is a UserVoice open for that request here that I highly recommend upvoting: https://twitch.uservoice.com/forums/310213-developers/suggestions/40616533-clips-eventsub-subscription-type-retitled

We're moving away from manual polling for Twitch events/data so we will wait for possible EventSub implementation. Marking this blocked for the time being.

**Comment by S1D3WYZ** [comment]
still blocked ?

**Comment by zunderscore** [comment]
> still blocked ?

Yes, this is still blocked. Twitch still has not added an event to notify us when a clip is created. We will update this issue when we have more to add.
